metricbeat/module/windows/perfmon/perfmon.go

```diff
@@ -63,6 +68,12 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 	if err := base.Module().UnpackConfig(&config); err != nil {
 		return nil, err
 	}
+	if metricsetName == base.Name() {
+		err := validateCounterConfig(config.CounterConfig)
+		if err != nil {
+			return nil, err
+		}
+	}
```

Comment:
Why is this change needed?

Reply:
the validateCounterConfig func will check if the instanceLabel has been set, the instance label is mandatory inside the pefrmon metricset but not in the webserver metricset

Reply:
Oh ok, I would prefer if we can make it more explicit, like with an additional internal flag in the metricset.

Could it happen that we create another metricset based on perfmon where the instance label is also mandatory?

Reply:
I have decided to leave the instance label optional in the perfmon metricset so no internal flag is required.